src/utils/address.cljs

```diff
@@ -6,7 +6,8 @@
 
 
 (def hex-prefix "0x")
-;; EIP-3770 is a format used by Status and described here: https://eips.ethereum.org/EIPS/eip-3770
+;; EIP-3770 is a format used by Status until v2.32 and described here:
+;; https://eips.ethereum.org/EIPS/eip-3770
 (def regx-eip-3770-address #"^(?:(?:eth:|arb1:|oeth:)(?=:|))*0x[0-9a-fA-F]{40}$")
```

Comment:
Given that we are removing support for these type of addresses, could we remove this regex?

Reply:
I wanted to remove that but I just kept it for backward compatibility. If a user A who will use v2.33 (w/o EIP-3770) tries to scan address of user B who is using v2.32. We need to recognise/accept that addresses on scanning/entering but strip chain prefixes on further steps. 

Maybe we can remove this regex in upcoming releases as it will give users time to move to newer versions completely. WDYT?

Reply:
I guess it's not needed as base chain integration is not released and the user will not see base prefix with this release due to this PR changes. Let me know your thoughts.